hybrid/pkg/agent/agent.go

```diff
@@ -0,0 +1,183 @@
+package hybrid_agent
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/hcl"
+	"github.com/hashicorp/hcl/hcl/printer"
+
+	"github.com/hewlettpackard/hybrid/pkg/common"
+	agentstorev1 "github.com/spiffe/spire-plugin-sdk/proto/spire/hostservice/server/agentstore/v1"
+	nodeattestorv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/agent/nodeattestor/v1"
+	configv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/service/common/config/v1"
+	"github.com/spiffe/spire/pkg/agent/plugin/nodeattestor/awsiid"
+	"github.com/spiffe/spire/pkg/agent/plugin/nodeattestor/azuremsi"
+	"github.com/spiffe/spire/pkg/agent/plugin/nodeattestor/gcpiit"
+	"github.com/spiffe/spire/pkg/agent/plugin/nodeattestor/k8spsat"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type HybridPluginAgent struct {
+	pluginList []common.Types
+	agentstorev1.UnimplementedAgentStoreServer
+	nodeattestorv1.UnsafeNodeAttestorServer
+	configv1.UnsafeConfigServer
+	logger      hclog.Logger
+	interceptor AgentInterceptorInterface
+	initStatus  error
```

Comment:
```suggestion
	agentstorev1.UnimplementedAgentStoreServer
	nodeattestorv1.UnsafeNodeAttestorServer
	configv1.UnsafeConfigServer
	
	pluginList []common.Types
	logger      hclog.Logger
	interceptor AgentInterceptorInterface
	initStatus  error
```

Implemented interfaces must go first, so it is easier to figure out you are implementing them

Reply:
done